docs/reference/env-variables.rst

```diff
@@ -0,0 +1,971 @@
+
+    * - | ``CL_KHR_FP64``
+        | Enable/Disable support for double precision.
+      - ``1``
+      - | 0: Disable
+        | 1: Enable
+
+    * - | ``CQ_THREAD_STACK_SIZE``
+        | The default command queue thread stack size.
+      - ``262144``
+      - | Unit: Byte 
+        | The default value corresponds to 256 kilobytes (KB). 
+
+    * - | ``CUDA_VISIBLE_DEVICES``
+        | Only devices whose index is present in the sequence are visible to HIP
+      - Unset by default.
+      - ``0,1,2``: Depending on the number of devices on the system.
+
+    * - | ``DEBUG_CLR_GRAPH_PACKET_CAPTURE``
+        | Enable/Disable graph packet capturing.
+      - ``0``
+      - | 0: Disable
+        | 1: Enable
+
+    * - | ``DEBUG_CLR_LIMIT_BLIT_WG``
+        | Limit the number of workgroups in blit operations.
+      - ``16``
+      -
+
+    * - | ``DISABLE_DEFERRED_ALLOC``
+        | Disables deferred memory allocation on device.
+      - ``0``
+      - | 0: Disable
+        | 1: Enable
+
+    * - | ``GPU_ADD_HBCC_SIZE``
+        | Add HBCC size to the reported device memory.
+      - ``0``
+      - | 0: Disable
+        | 1: Enable
+
+    * - | ``GPU_ANALYZE_HANG``
+        | Enables GPU hang analysis.
+      - ``0``
+      - | 0: Disable
+        | 1: Enable
+
+    * - | ``GPU_BLIT_ENGINE_TYPE``
+        | Blit engine type.
+      - ``0``
+      - | 0: Default
+        | 1: Host
+        | 2: CAL
+        | 3: Kernel
+
+    * - | ``GPU_CP_DMA_COPY_SIZE``
+        | Set maximum size of CP DMA copy in kB.
+      - ``1``
+      - Unit: kilobyte (KB)
+
+    * - | ``GPU_DEBUG_ENABLE``
+        | Enables collection of extra info for debugger at some performance cost.
+      - ``0``
+      - | 0: Disable
+        | 1: Enable
+
+    * - | ``GPU_DEVICE_ORDINAL``
+        | Select the device ordinal, a comma separated list of available devices.
+      - Unset by default.
+      - ``0,2``: Expose the 1. and 3. device in the system.
+
+    * - | ``GPU_DUMP_BLIT_KERNELS``
+        | Dump the kernels for blit manager.
+      - ``0``
+      - | 0: Disable
+        | 1: Enable
+
+    * - | ``GPU_DUMP_CODE_OBJECT``
+        | Dump code object.
+      - ``0``
+      - | 0: Disable
+        | 1: Enable
+
+    * - | ``GPU_ENABLE_COOP_GROUPS``
+        | Enables cooperative group launch.
+      - ``1``
+      - | 0: Disable
+        | 1: Enable
+
+    * - | ``GPU_ENABLE_HW_P2P``
+        | Enables HW P2P path.
+      - ``0``
+      - | 0: Disable
+        | 1: Enable
+
+    * - | ``GPU_ENABLE_LC``
+        | Enables LC path.
+      - ``1``
+      - | 0: Disable
+        | 1: Enable
+
+    * - | ``GPU_ENABLE_PAL``
+        | Enables PAL backend.
+      - ``2``
+      - | 0: ROC
+        | 1: PAL
+        | 2: ROC or PAL
+
+    * - | ``GPU_ENABLE_WAVE32_MODE``
+        | Enables Wave32 compilation in HW if available.
+      - ``1``
+      - | 0: Disable
+        | 1: Enable
+
+    * - | ``GPU_ENABLE_WGP_MODE``
+        | Enables WGP Mode in HW if available.
```

Comment:
```suggestion
        | Enables WGP Mode in hardware, if available.
```
Specify WGP full form if possible

Reply:
Added WGP description. @SwRaw please check.

Reply:
I meant to add full forms for all abbreviations in the document (if you can get that information).